src/proc/color-formats-converter.cpp

```diff
@@ -354,6 +354,195 @@ namespace librealsense
+                    b[4], g[4], r[4], b[5], g[5], r[5],
+                    b[6], g[6], r[6], b[7], g[7], r[7],
+                    b[8], g[8], r[8], b[9], g[9], r[9],
+                    b[10], g[10], r[10], b[11], g[11], r[11],
+                    b[12], g[12], r[12], b[13], g[13], r[13],
+                    b[14], g[14], r[14], b[15], g[15], r[15],
+                };
+                librealsense::copy(*dst, out, sizeof out);
+                *dst += sizeof out;
+                continue;
+            }
+
+            if (FORMAT == RS2_FORMAT_RGBA8)
+            {
+                uint8_t out[16 * 4] = {
+                    r[0], g[0], b[0], 255, r[1], g[1], b[1], 255,
+                    r[2], g[2], b[2], 255, r[3], g[3], b[3], 255,
+                    r[4], g[4], b[4], 255, r[5], g[5], b[5], 255,
+                    r[6], g[6], b[6], 255, r[7], g[7], b[7], 255,
+                    r[8], g[8], b[8], 255, r[9], g[9], b[9], 255,
+                    r[10], g[10], b[10], 255, r[11], g[11], b[11], 255,
+                    r[12], g[12], b[12], 255, r[13], g[13], b[13], 255,
+                    r[14], g[14], b[14], 255, r[15], g[15], b[15], 255,
+                };
+                librealsense::copy(*dst, out, sizeof out);
+                *dst += sizeof out;
+                continue;
+            }
+
+            if (FORMAT == RS2_FORMAT_BGRA8)
+            {
+                uint8_t out[16 * 4] = {
+                    b[0], g[0], r[0], 255, b[1], g[1], r[1], 255,
+                    b[2], g[2], r[2], 255, b[3], g[3], r[3], 255,
+                    b[4], g[4], r[4], 255, b[5], g[5], r[5], 255,
+                    b[6], g[6], r[6], 255, b[7], g[7], r[7], 255,
+                    b[8], g[8], r[8], 255, b[9], g[9], r[9], 255,
+                    b[10], g[10], r[10], 255, b[11], g[11], r[11], 255,
+                    b[12], g[12], r[12], 255, b[13], g[13], r[13], 255,
+                    b[14], g[14], r[14], 255, b[15], g[15], r[15], 255,
+                };
+                librealsense::copy(*dst, out, sizeof out);
+                *dst += sizeof out;
+                continue;
+            }
+
+            
+        }
+    }
+
+    /////////////////////////////
+    // M420 unpacking routines //
+    /////////////////////////////
+    // This templated function unpacks M420 into Y8/Y16/RGB8/RGBA8/BGR8/BGRA8, depending on the compile-time parameter FORMAT.
+    // It is expected that all branching outside of the loop control variable will be removed due to constant-folding.
+    // The M420 is a standard format - see: https://www.kernel.org/doc/html/v4.10/media/uapi/v4l/pixfmt-m420.html
+    // Its configuration is as following: 2 lines of Y then one line of UV (line size is width)
+    // There is one Y value for each pixel, and one pair of U,V values for 4 pixels. 
+    // For example: for the first 3 lines of the frame:
+    // Y0  Y1   Y2   Y3   ....
+    // Yw  Yw+1 Yw+2 Yw+3 .... (Yw:Ywidth)
```

Comment:
What does the `w` in `Yw` stands for?

Reply:
w: width (as written in the comment)
Yw is y value for index w, seems readable to me - if you think about another way to write it, I have no problem to change it